docs/streams/developer-guide/dsl-api.html

```diff
@@ -1891,11 +1891,6 @@ <h4 class="anchor-heading"><a id="streams_concepts_globalktable" class="anchor-l
                                         </li>
                                         <li><p class="first">The join will be triggered under the conditions listed below whenever new input is received.  When it is triggered, the user-supplied <code class="docutils literal"><span class="pre">ValueJoiner</span></code> will be called to produce
                                             join output records.</p>
-                                            <blockquote>
-                                                <div><ul class="simple">
-                                                    <li>Input records with a <code class="docutils literal"><span class="pre">null</span></code> value are ignored and do not trigger the join.</li>
```

Comment:
For me, this line is still accurate after the feature KAFKA-12317. If I am not mistaken, It's records with null keys (and non null value) which are no longer ignored